client/webserver/site/src/html/login.tmpl

```diff
@@ -8,6 +8,8 @@
         <div class="pb-2">
           <label for="pw" class="pl-1 mb-1">Password</label>
           <input type="password" class="form-control select" id="pw" autocomplete="current-password">
+          <label for="rememberPass" class="pl-1 mb-1">Remember my password</label>
+          <input type="checkbox" id="rememberPass">
```

Comment:
To be consistent with other forms, put the input before the label, and add `class="form-check-input"` to the checkbox. I think you'll need to wrap everything in a `class="pl-4"` div too, but not certain.

Also looks like you need some padding between the password field and the remember-password input.

![image](https://user-images.githubusercontent.com/6109680/128614260-567c4fee-7dc9-4da5-a0f8-11d7c3627918.png)

Reply:
Looks way better now.